tests/cmdline/commands/test_devel.py

```diff
@@ -8,12 +8,51 @@
 # For further information please visit http://www.aiida.net               #
 ###########################################################################
 """Tests for ``verdi devel``."""
+from plumpy.process_comms import RemoteProcessThreadController
+from plumpy.process_states import ProcessState
+import pytest
+
 from aiida.cmdline.commands import cmd_devel
-from aiida.orm import Node
+from aiida.engine import submit
+from aiida.orm import Int, Node
 
 
 def test_run_sql(run_cli_command):
     """Test ``verdi devel run-sql``."""
     options = ['SELECT COUNT(*) FROM db_dbnode;']
     result = run_cli_command(cmd_devel.devel_run_sql, options)
     assert str(Node.collection.count()) in result.output, result.output
+
+
+@pytest.mark.usefixtures('stopped_daemon_client')
+def test_revive_without_daemon(run_cli_command):
+    """Test that ``verdi devel revive`` fails if no daemon is running."""
+    assert run_cli_command(cmd_devel.devel_revive, raises=True)
+
+
+@pytest.mark.usefixtures('aiida_profile', 'started_daemon_client')
+def test_revive(run_cli_command, monkeypatch, aiida_local_code_factory, submit_and_await):
+    """Test that ``verdi devel revive`` fails if no daemon is running."""
```

Comment:
```suggestion
    """Test that ``verdi devel revive`` works, when encountering lost RabbitMQ task."""
```

Reply:
Given the separate test of `test_revive`, this test could probably be recudec in scope.